google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -234,7 +251,26 @@ public void run() {
                 appendLoop();
               }
             });
+
+    this.reportingThread =
+        new Thread(
+            new Runnable() {
+              @Override
+              public void run() {
+                while (true) {
+                  reportSingleCurrentStatus();
+                }
+              }
+            });
     this.appendThread.start();
+    this.reportingThread.start();
+  }
+
+  private void reportSingleCurrentStatus() {
```

Comment:
Is this too much logging? Is it going to be in the final code?

Reply:
Not going to be in the final code, it's reported every 5 seconds